### PR TITLE
zulu 11.2.3:11.0.1

### PR DESCRIPTION
**Pull request by mistydemeo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

This version of the package doesn't include a `+` in the name, so I've removed the comma from the version and the URL. I kept the colon in the version for consistency with what the package did before and the other Zulu packages, and in case the `+` version comes back.

### CODE REVIEW

**Comment by commitay** [comment]
👋 Nice to see you in this part of Homebrew @mistydemeo!

>  kept the colon in the version for consistency with what the package did before and the other Zulu packages, and in case the + version comes back.

We might need to use the colon again for future versions but can you change this to use comma instead of colon for now please. We prefer to only use it when it's needed.

File: Casks/zulu.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'zulu' do
-  version '10.3,5:10.0.2'
-  sha256 'dfc5a9cae3fbfb45d565bcd067829b7273704c976ff00a63794c9f9742fe4f76'
+  version '11.2.3:11.0.1'
```
Comment (vitorgalvao) [inline Casks/zulu.rb]:
```suggestion
  version '11.2.3,11.0.1'
```

File: Casks/zulu.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  sha256 '5e2364ae8e1f4d81b2253de3d52dee5701f8f7c31160fac2d23b4f1c371cff2b'
 
-  url "https://cdn.azul.com/zulu/bin/zulu#{version.before_comma}+#{version.after_comma.before_colon}-jdk#{version.after_colon}-macosx_x64.dmg",
+  url "https://cdn.azul.com/zulu/bin/zulu#{version.before_colon}-jdk#{version.after_colon}-macosx_x64.dmg",
```
Comment (vitorgalvao) [inline Casks/zulu.rb]:
```suggestion
  url "https://cdn.azul.com/zulu/bin/zulu#{version.before_comma}-jdk#{version.after_comma}-macosx_x64.dmg",
```

File: Casks/zulu.rb [hunk 3]
```diff
@@ -11,10 +11,10 @@
 
   postflight do
     system_command '/bin/mv',
-                   args: ['-f', '--', "/Library/Java/JavaVirtualMachines/zulu-#{version.major}.jdk", "/Library/Java/JavaVirtualMachines/zulu-#{version.before_comma}+#{version.after_comma.before_colon}.jdk"],
+                   args: ['-f', '--', "/Library/Java/JavaVirtualMachines/zulu-#{version.major}.jdk", "/Library/Java/JavaVirtualMachines/zulu-#{version.before_colon}.jdk"],
```
Comment (vitorgalvao) [inline Casks/zulu.rb]:
```suggestion
                   args: ['-f', '--', "/Library/Java/JavaVirtualMachines/zulu-#{version.major}.jdk", "/Library/Java/JavaVirtualMachines/zulu-#{version.before_comma}.jdk"],
```

File: Casks/zulu.rb [hunk 4]
```diff
@@ -11,10 +11,10 @@
...
                    sudo: true
     system_command '/bin/ln',
-                   args: ['-nsf', '--', "/Library/Java/JavaVirtualMachines/zulu-#{version.before_comma}+#{version.after_comma.before_colon}.jdk", "/Library/Java/JavaVirtualMachines/zulu-#{version.major}.jdk"],
+                   args: ['-nsf', '--', "/Library/Java/JavaVirtualMachines/zulu-#{version.before_colon}.jdk", "/Library/Java/JavaVirtualMachines/zulu-#{version.major}.jdk"],
```
Comment (vitorgalvao) [inline Casks/zulu.rb]:
```suggestion
                   args: ['-nsf', '--', "/Library/Java/JavaVirtualMachines/zulu-#{version.before_comma}.jdk", "/Library/Java/JavaVirtualMachines/zulu-#{version.major}.jdk"],
```

File: Casks/zulu.rb [hunk 5]
```diff
@@ -26,7 +26,7 @@
 
   uninstall pkgutil: "com.azulsystems.zulu.#{version.major}",
             delete:  [
-                       "/Library/Java/JavaVirtualMachines/zulu-#{version.before_comma}+#{version.after_comma.before_colon}.jdk",
+                       "/Library/Java/JavaVirtualMachines/zulu-#{version.before_colon}.jdk",
```
Comment (vitorgalvao) [inline Casks/zulu.rb]:
```suggestion
                       "/Library/Java/JavaVirtualMachines/zulu-#{version.before_comma}.jdk",
```

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
As @commitay mentioned, we prefer comma first and colon only when comma is in use.

Taking the opportunity to try out the suggestions feature.

**Comment by mistydemeo** [comment]
> Taking the opportunity to try out the suggestions feature.

Good idea! Looks like it makes it one commit per hunk, which is kind of annoying, so I rebased your suggestions down into a single commit.

**Comment by commitay** [comment]
Thanks @mistydemeo!

**Pull request merged**